hardhat.config.ts

```diff
@@ -72,12 +74,21 @@ const config: HardhatUserConfig = {
     },
     arbitrumRinkeby: {
       url: process.env.ARBITRUM_TESTNET_URL,
-      accounts: [`${process.env.ARBITRUM_MAINNET_PK}`],
+      accounts: [`${privateKey}`],
```

Comment:
Recommend reverting this to not wreck people's repo setups

Reply:
Reverting.